cellprofiler/__main__.py

```diff
@@ -78,23 +80,27 @@ def main(args=None):
                 a.write(line)
```

Comment:
Changed 'with' loop to the following to deal with "IOError: [Errno 13] Permission denied" error; see https://stackoverflow.com/a/23212515
```
for line in downloaded_pipeline:
    temp_pipe_file.write(line)
```